src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -118,6 +136,9 @@ SHAMapStoreImp::SHAMapStoreImp(
 
     get_if_exists(section, "online_delete", deleteInterval_);
 
+    // Always initialize state database for pinned ranges persistence
```

Comment:
TODO: 

Is this actually appropriate for standalone mode? 
Unittests? 
Where is it made?